tasks/upload_processor.py

```diff
@@ -322,6 +339,7 @@ def process_impl_within_lock(
                     )
             else:
                 with metrics.timer(f"{self.metrics_prefix}.save_report_results"):
```

Comment:
I wonder if changing this for the sentry trace would be a good idea

Reply:
I already added a tracing annotation. will remove this metric then.